pom.xml

```diff
@@ -45,6 +45,19 @@ under the License.
   <url>https://maven.apache.org/ref/${project.version}/</url>
   <inceptionYear>2001</inceptionYear>
 
+  <repositories>
+    <repository>
+      <id>repository.apache.org</id>
+      <url>https://repository.apache.org/content/repositories/snapshots</url>
+      <releases>
+        <enabled>false</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+    </repository>
+  </repositories>
```

Comment:
There is in ASF pom ....
https://github.com/apache/maven-apache-parent/blob/master/pom.xml#L110-L129
Is it not the same?

Reply:
eh, missed it :) I assumed maven build does not allow snapshots for some reason (and CI builds w/ snapshots would fail)

Reply:
Now I remember what I mixed up: not snapshot artifacts, but **staged artifacts**....